admin/templates/configuration/etc/log4cxx.worker.properties

```diff
@@ -2,9 +2,9 @@
 # Configuration file for log4cxx
 #
 
-log4j.rootLogger=DEBUG, XRDLOG
+log4j.rootLogger=INFO, CONSOLE
```

Comment:
OK, that was the initial recommendation. If I change INFO to DEBUG, then I should change the test whether or not those messages go through. I assume the call will be isDebugEnabled() (at least that is what I get from log4cxx docs). It would be good if you confirm.